franklin/github/views.py

```diff
@@ -89,11 +89,11 @@ def register_repo(request):
     responseMessages:
         - code: 400
           message: Invalid json received or Bad Request from Github
+        - code: 403
+          message: Current user does not have deployment permissions
         - code: 422
           message: Validation error from Github
     """
```

Comment:
The registration endpoint now requires admin level permissions for a repo. We don't "need" this yet, but we will when we generate a deploy key. Builder will need that to properly clone repos and only admins of a repo can add new deploy keys.

Keep that in mind if you are testing API and are getting an error for a test project that used to work.